spec/support/factories/users.rb

```diff
@@ -1,7 +1,9 @@
 FactoryGirl.define do
-  factory :user, aliases: [:author, :editor], class: Basechurch::User do
+  factory :user, aliases: [:author, :editor] do
     name Forgery(:name).full_name
     email { Forgery(:email).address }
     password Forgery(:lorem_ipsum).words(2)
+    uid { Forgery('basic').number(at_least: 1000, at_most: 9999).to_s }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.